services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -428,6 +437,76 @@ async def pull_input_ports(
     return transferred_bytes
 
 
+@containers_router.post(
+    "/containers/directory-watcher:disable",
+    summary="Disable directory-watcher event propagation",
+    response_model=None,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def disable_directory_watcher(
+    app: FastAPI = Depends(get_application),
+) -> Response:
+    directory_watcher.disable_directory_watcher(app)
+    # SEE https://github.com/tiangolo/fastapi/issues/2253
+    return Response(status_code=status.HTTP_204_NO_CONTENT)
+
+
+@containers_router.post(
+    "/containers/directory-watcher:enable",
+    summary="Enable directory-watcher event propagation",
+    response_model=None,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def enable_directory_watcher(app: FastAPI = Depends(get_application)) -> Response:
+    directory_watcher.enable_directory_watcher(app)
+    # SEE https://github.com/tiangolo/fastapi/issues/2253
+    return Response(status_code=status.HTTP_204_NO_CONTENT)
+
+
+@containers_router.post(
```

Comment:
https://google.aip.dev/121

Reply:
in the end I went with `POST /containers/ports/outputs/dirs` as it seemed the better alternative